### ISSUE TITLE
nil values and item_order.food.name and item_order

### ISSUE THREAD

**Issue by bfellerath** [bfellerath/RESTaurant#2]
I am using binding pry to see how to get to the values I want to but I haven't been able to yet. I wrote down what I want to accomplish today, Rafa had said Item_order.food.name might work but I am having trouble getting that to work.

**Comment by rapala61** [comment]
Try `Item_order.food_item.name` Your model for foods is name food_items so that might be the discrepancy, the point of that syntax is to access a food item from a item_order since item order holds the a belongs_to association with food_item.

Also, in your model item_order, remember that the `belongs_to` keyword should point to the singular of the model not the plural, you have to edit your `line #2` to reflect that.

**Comment by bfellerath** [comment]
Yes, I tried Item_order.food_item.name, sorry I typed it wrong above.

I changed both line #2 and #3 (they were both plural) and I am testing that now.

**Comment by bfellerath** [comment]
Okay I changed it to `item_order.food_item.name` 

When I load the index page now I get an error: undefined method 'name' for nil class which means food_item. I've tried changing it to "food_items" I know that wouldn't work. It gave me a different error. But I did try that.

 At this point I'm just going to keep trying different things. I feel like I am moving really slowly on this, Doing some googling etc. I am hoping at some point I see a pattern here.
